### PR TITLE
Chat Text Ui

### LINKED ISSUE #360: Chat screen UI enhancement
* Add names on chat bubbles
* Fix the OulinedTextField

### PR DESCRIPTION
**Pull request by Zep3k**
Screenshot_20240521-131342_SkySync
Close #360

### CODE REVIEW

File: app/src/main/java/ch/epfl/skysync/components/ChatText.kt [hunk 1]
```diff
@@ -110,35 +105,32 @@ fun ChatBubble(message: ChatMessage, index: String) {
         }
       }
   Row(
```
Comment (RAFInery) [inline app/src/main/java/ch/epfl/skysync/components/ChatText.kt:107]: why is there a Row composable if it contains only one composable? For the alignment?

File: app/src/main/java/ch/epfl/skysync/components/ChatText.kt [hunk 2]
```diff
@@ -110,35 +105,32 @@ fun ChatBubble(message: ChatMessage, index: String) {
...
-              text = messageContent,
-              color = contentColor,
-              modifier = Modifier.padding(bottom = 2.dp).testTag("ChatBubbleMessage$index"))
-          Spacer(modifier = Modifier.size(4.dp))
-          Text(
-              text = time,
-              color = Color.Gray,
-              fontSize = 9.sp,
-              modifier = Modifier.align(Alignment.Bottom).testTag("ChatBubbleTime$index"))
+          Column(
```
Comment (RAFInery) [inline app/src/main/java/ch/epfl/skysync/components/ChatText.kt:118]: could be modularized into smaller composables to increase readability

**Review by RAFInery — APPROVED** [review]
LGTM

File: app/src/main/java/ch/epfl/skysync/components/ChatText.kt (hunk 1 above)
Comment (Zep3k) [reply app/src/main/java/ch/epfl/skysync/components/ChatText.kt:107]: yes

**Pull request merged**